docs/source/onedal/algorithms/kernel-functions/sigmoid-kernel.rst

```diff
@@ -0,0 +1,57 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. default-domain:: cpp
+
+.. _alg_sigmoid_kernel:
+
+===================================
+Sigmoid kernel
+===================================
+
+.. include::  ../../../includes/kernel-functions/sigmoid-kernel-introduction.rst
+
+------------------------
+Mathematical formulation
+------------------------
+
+.. _sigmoid_kernel_c_math:
+
+Computing
+---------
+
+Given a set :math:`X` of :math:`n` feature vectors :math:`x_1 = (x_{11}, \ldots, x_{1p}), \ldots, x_n = (x_{n1}, \ldots, x_{np})`
+of dimension :math:`p` and a set :math:`Y` of :math:`m`
+feature vectors :math:`y_1 = (y_{11}, \ldots, y_{1p}), \ldots, y_m = (y_{m1}, \ldots, x_{mp})`,
+the problem is to compute the sigmoid kernel function :math:`K(x_i, y_j)` for any pair of input vectors:
+
+.. math::
+   K(x_i, y_j) = tanh(k {x_i}^T y_j + b),
```

Comment:
could you please check the output for this equation?

I believe this is the correct way:

```suggestion
   K(x_i, y_j) = \tanh(k {x_i}^T y_j + b),
```

Reply:
I've checked. You're right